shared/src/business/useCaseHelper/trialSessions/setNoticeOfChangeToRemoteProceeding.js

```diff
@@ -0,0 +1,146 @@
+const {
+  aggregatePartiesForService,
+} = require('../../utilities/aggregatePartiesForService');
+const {
+  CASE_STATUS_TYPES,
+  DOCUMENT_PROCESSING_STATUS_OPTIONS,
+  SYSTEM_GENERATED_DOCUMENT_TYPES,
+  TRIAL_SESSION_PROCEEDING_TYPES,
+} = require('../../entities/EntityConstants');
+const { DocketEntry } = require('../../entities/DocketEntry');
+
+const serveNoticesForCase = async (
+  applicationContext,
+  {
+    caseEntity,
+    newPdfDoc,
+    noticeDocketEntryEntity,
+    noticeDocumentPdfData,
+    PDFDocument,
+    servedParties,
+  },
+) => {
+  await applicationContext.getUseCaseHelpers().sendServedPartiesEmails({
+    applicationContext,
+    caseEntity,
+    docketEntryId: noticeDocketEntryEntity.docketEntryId,
+    servedParties,
+  });
+
+  if (servedParties.paper.length > 0) {
+    const noticeDocumentPdf = await PDFDocument.load(noticeDocumentPdfData);
+
+    await applicationContext
+      .getUseCaseHelpers()
+      .appendPaperServiceAddressPageToPdf({
+        applicationContext,
+        caseEntity,
+        newPdfDoc,
+        noticeDoc: noticeDocumentPdf,
+        servedParties,
+      });
+  }
+};
+
+/**
+ * setNoticeOfChangeToRemoteProceeding
+ *
+ * @param {object} providers the providers object
+ * @param {object} providers.applicationContext the application context
+ * @param {object} providers.caseEntity the case data
+ * @param {object} providers.currentTrialSession the old trial session data
+ * @param {object} providers.newTrialSessionEntity the new trial session data
+ * @param {object} providers.userId the user ID
+ * @returns {object} the created trial session
+ */
+exports.setNoticeOfChangeToRemoteProceeding = async (
```

Comment:
This method makes excellent use of helper methods, it's extremely easy to read and understand high level what's going on. Nice job!